src/api/search.api.js

```diff
@@ -132,7 +132,7 @@ let cancelToken = null
  *   returned
  * @returns {Promise<CombinedSearchResults>}
  */
-async function search(queryString = '', lang = '', outputProjection) {
+async function search(queryString = '', lang = '', outputProjection = DEFAULT_PROJECTION) {
```

Comment:
I would not used any default projection in the api functions to avoid surprise when changing between projection.

Reply:
then I have to change the function signature, as it's the last param